public/app/features/logs/logsModel.ts

```diff
@@ -640,11 +639,23 @@ const updateLogsVolumeConfig = (
 };
 
 type LogsVolumeQueryOptions<T extends DataQuery> = {
-  extractLevel: (dataFrame: DataFrame) => LogLevel;
+  extractLevel?: (dataFrame: DataFrame) => LogLevel;
```

Comment:
Just to confirm, what is the decision related to logs volume query options and level overrided:
- we don't need it for now
- we want to have it as function
- instead of function, we want to just have the option to specify the name of `level` field (e.g. somewhere it can be `error_level`, or `LVL`, or `LEVEL`  or anything else?

Reply:
Both `we don't need it for now` and `option to specify the level field` works for me. Now that we're at it, we can easily add support to specify the `level` field and win-win.